src/tester.rs

```diff
@@ -422,42 +425,62 @@ fn run_tests(
                     None => continue,
                 };
                 let mut meant_to_error = false;
+                // First, see which of the tests passed.
```

Comment:
If I understand correctly, each execution of this closure runs one test?

```
for p in test_files {
    ...
    pool.execute(...
```

If so, I'm a little confused by this comment. Don't we know which test passed/failed due to the construction of the test loop?

Maybe I've misunderstood?

Reply:
Ohhhh. You mean find out *how* it failed (out of {exit status, stderr match, stdout match}), not *which* test failed. I think we can tweak this comment to make it clearer.

Reply:
OK will tweak.